app/Http/Controllers/Api/DepartmentController.php

```diff
@@ -39,7 +39,7 @@ public function create(Request $request)
         $department->id = $id;
         $department->name = $departmentName;
         $department->code = $departmentCode;
-        $department->school_id = $this->schoolId;
+        //$department->school_id = $this->schoolId;
```

Comment:
where is the school id coming from?

Reply:
we don't need that statment anymore. However used to create  "$this->schoolI"  earlier from a constructor 😕